manifests/init.pp

```diff
@@ -76,7 +78,7 @@
   $plugin_dir = "${extensions_dir}/plugins"
 
   $installdir = "${installroot}/${service}"
-  $tmpzip = "/usr/local/src/${package_name}-${version}.zip"
+  $tmpzip = "${download_dir}/${package_name}-${version}.zip"
```

Comment:
@faxm0dem Instead of adding a new parameter, how about setting `$tmpzip` to `/tmp/${package_name}-${version}.zip`?